src/libvirtApi/domain.js

```diff
@@ -865,6 +865,36 @@ export function domainSetCpuMode({
     ], opts);
 }
 
+export function domainSetTitle({
```

Comment:
domainSetTitle and domainSetDescription have essentially the same function body. Can you maybe parameterize it?

Reply:
Yes I will do it, no problem. Acually I thought the same, but tried to match the rest of the code and doing as little changes as possible.  All other field seems to have their own functions and there are a lot cockpit.spawn methods which call virt-xml with basically the same arguments and I didn't want to refactor a whole file on my first contribution ;-).